From 4bec99174a5911457cddd87ceab4f5e038cc8141 Mon Sep 17 00:00:00 2001
From: Wolfram Sang <w.sang@pengutronix.de>
Date: Sun, 29 Aug 2010 18:17:14 +0200
Subject: USB: core: update comment to match current function name
Patch-mainline: v2.6.37
Git-commit: 4bec99174a5911457cddd87ceab4f5e038cc8141

Found while debugging a USB problem and trying to find the mentioned function.

Signed-off-by: Wolfram Sang <w.sang@pengutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/hub.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -3696,7 +3696,7 @@ static int usb_reset_and_verify_device(s
 	}
 
 	if (!parent_hdev) {
-		/* this requires hcd-specific logic; see OHCI hc_restart() */
+		/* this requires hcd-specific logic; see ohci_restart() */
 		dev_dbg(&udev->dev, "%s for root hub!\n", __func__);
 		return -EISDIR;
 	}
